### ISSUE TITLE
 Issues in the rendered html

### ISSUE THREAD

**Issue by minarainbow** [arXiv/html_feedback#1794]
### Description

the description command is shown to the HTML.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

16923a06-0949-46da-84f4-db3455ca046e

### Paper URL

https://arxiv.org/html/2407.17681v1

### Browser

Chrome/125.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
